File: certificates/problem-solving-intermediate/file-renaming/stub.cc
#include <bits/stdc++.h>

using namespace std;



/*
 * Complete the 'renameFile' function below.
 *
 * The function is expected to return an INTEGER.
 * The function accepts following parameters:
 *  1. STRING newName
 *  2. STRING oldName
 */

int renameFile(string newName, string oldName) {

}

int main()
{
    ofstream fout(getenv("OUTPUT_PATH"));

    string newName;
    getline(cin, newName);

    string oldName;
    getline(cin, oldName);

    int result = renameFile(newName, oldName);

    fout << result << "\n";

    fout.close();

    return 0;
}

File: certificates/problem-solving-intermediate/file-renaming/stub.py
#!/bin/python3

import math
import os
import random
import re
import sys



#
# Complete the 'renameFile' function below.
#
# The function is expected to return an INTEGER.
# The function accepts following parameters:
#  1. STRING newName
#  2. STRING oldName
#

def renameFile(newName, oldName):
    # Write your code here
    pass

if __name__ == '__main__':
    fptr = open(os.environ['OUTPUT_PATH'], 'w')

    newName = input()

    oldName = input()

    result = renameFile(newName, oldName)

    fptr.write(str(result) + '\n')

    fptr.close()
